docs/mcmc/index.rst

```diff
@@ -117,39 +120,41 @@ References
 Example
 =======
 
-.. note::
-
-   This example probably needs to be simplified to reduce the run time
```

Comment:
How long does this take? If I remember it's not a particularly fast operation, which was one of the reasons why I'd not bothered to auto-generate the results.

Reply:
If I copy out the example to just run the code (and not build the html on sphinx which needs to be done anyway), it's about 10 s on my machine.

Reply:
I thought it was slower than that. Your machine is likely a little-bit beefier than the one that RTD uses (or I developed the example on ;-).

Reply:
[that's assuming these times can be meaningfully compared, which as we've just been discussing with the cache code, is not always true ;-]